_projects/2024/100543010/100543010.Rmd

```diff
@@ -0,0 +1,897 @@
+---
+title: "Key Global Partners in U.S. Trade for 2023"
+description: |
+  Worldwide U.S. Trade in Billions by country in 2023
+categories: '2024'
+author: Jason Brunner
+date: 2024-12-16
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+    toc_float: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, fig.width= 14, fig.height= 14, 
```

Comment:
Your graph has more width than height, right? Because I see a lot of white space around your plots right now.